### PR TITLE
Analytics + more bug fixes

### PR DESCRIPTION
**Pull request by Geometrically**
Fixes MOD-466

### CODE REVIEW

**Review by thesuzerain — APPROVED** [review]

File: theseus_gui/src/components/RowDisplay.vue [hunk 1]
```diff
@@ -114,20 +109,17 @@ const handleInstanceRightClick = (event, passedInstance) => {
...
-      await args.item.stop()
+      await args.item.stop(null, 'InstanceRowContextMenu')
       break
     case 'add_content':
       await args.item.addContent()
       break
     case 'edit':
       await args.item.seeInstance()
       break
-    case 'delete':
```
Comment (CodexAdrian) [inline theseus_gui/src/components/RowDisplay.vue:128]: Same thing here

File: theseus_gui/src/main.js [hunk 2]
```diff
@@ -17,6 +17,16 @@ app.use(pinia)
 app.use(FloatingVue)
 app.mixin(loadCssMixin)
 
+// FIXME: this causes the app to lag
```
Comment (CodexAdrian) [inline theseus_gui/src/main.js:20]: was this meant to be uncommented?

File: theseus_gui/src/pages/instance/Index.vue [hunk 3]
```diff
@@ -249,33 +239,25 @@ const handleOptionsClick = async (args) => {
...
-    case 'delete':
-      await remove(instance.value.path).catch(handleError)
-      break
     case 'open_folder':
       await showInFolder(instance.value.path)
       break
     case 'copy_path':
       await navigator.clipboard.writeText(instance.value.path)
       break
-    case 'open_link':
```
Comment (CodexAdrian) [inline theseus_gui/src/pages/instance/Index.vue:264]: duplicate of above, this should still be an option

**Review by CodexAdrian — CHANGES_REQUESTED** [review]
Mostly looks good, just curious as to why so many context menu options were removed

**Pull request merged**